docs/docs/how-tos/develop-local-packages.md

```diff
@@ -0,0 +1,66 @@
+---
+id: develop-local-packages
+title: Develop Local Packages on Nebari
+description: How to install a dev package from a local project
+---
+
+# How to Develop Local Packages on Nebari
+
+As a software development platform, Nebari can be used for active package development. Since `conda-store` manages all environments, a dynamic installation of a local package requires additional care (after all, `pip install -e .` is explicitly disallowed for various reasons).
```

Comment:
```suggestion
As a software development platform, Nebari can be used for active package development. Since `conda-store` manages all environments, a dynamic installation of a local package requires additional care (`pip install -e .` is explicitly disallowed for various reasons, see below).
```